front-end/src/components/Post/Post.tsx

```diff
@@ -112,12 +112,16 @@ const Post = ( { className, data, isMotion = false, isProposal = false, isRefere
 		postStatus = treasuryPost?.onchain_link?.onchain_treasury_spend_proposal?.[0]?.treasuryStatus?.[0].status;
 	}
 
+	const proposalAddressesIncludesProposer = isProposal && proposalPost?.onchain_link?.proposer_address && addresses?.includes(proposalPost.onchain_link.proposer_address);
```

Comment:
I'd change the naming for something more straightforward (Same for the others: `isReferendumProposer`, `isMotionProposer`, `isTreasuryProposer`.):
```suggestion
	const isProposalProposer = isProposal && proposalPost?.onchain_link?.proposer_address && addresses?.includes(proposalPost.onchain_link.proposer_address);
```